libs/sql-schema-describer/src/walkers.rs

```diff
@@ -183,9 +183,9 @@ impl<'a, 'schema> ForeignKeyWalker<'schema> {
 }
 
 pub struct IndexWalker<'a> {
-    schema: &'a SqlSchema,
-    table: &'a Table,
-    index: &'a Index,
+    pub schema: &'a SqlSchema,
+    pub table: &'a Table,
+    pub index: &'a Index,
```

Comment:
These should be private, but it's something I can fix in a subsequent PR, let's not block the merge on this.